Description: update udev script to work in Debian
Author: Valentin Vidic <Valentin.Vidic@CARNet.hr>
Last-Update: 2018-04-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/gfs2/scripts/gfs2_withdraw_helper
+++ b/gfs2/scripts/gfs2_withdraw_helper
@@ -19,7 +19,7 @@
 
 if [ -z "$DM_DEV" ]
 then
-    /usr/bin/dmsetup suspend $DM_DEV
+    /sbin/dmsetup suspend $DM_DEV
 fi
 
 # Signal completion of withdraw
